internal/ingress/controller/template/template.go

```diff
@@ -729,12 +729,12 @@ func buildOpentracingLoad(input interface{}) string {
 		return ""
 	}
 
-	buf := bytes.NewBufferString("load_module/etc/nginx/modules/ngx_http_opentracing_module.so;")
+	buf := bytes.NewBufferString("load_module /etc/nginx/modules/ngx_http_opentracing_module.so;")
 
 	if cfg.ZipkinCollectorHost != "" {
-		buf.WriteString("load_module/etc/nginx/modules/ngx_http_zipkin_module.so;")
+		buf.WriteString("load_module /etc/nginx/modules/ngx_http_zipkin_module.so;")
```

Comment:
Is this missing a newline as well? Not critical but for readability's sake.

Reply:
Interestingly, no other buf.WriteString() here have an "\n" in the end, yet the formatting seems fine.